"""
    VIER Cognitive Voice Health API

    CVG health  # noqa: E501

    The version of the OpenAPI document: 1.24.4
    Contact: support@vier.ai
    Generated by: https://openapi-generator.tech
"""


import sys
import unittest

import cvg_sdk
from cvg_sdk.model.audio_playback_failed import AudioPlaybackFailed
from cvg_sdk.model.endpoint_call import EndpointCall
from cvg_sdk.model.endpoint_call_failed import EndpointCallFailed
from cvg_sdk.model.health_event_status import HealthEventStatus
from cvg_sdk.model.internal_error import InternalError
from cvg_sdk.model.internal_error_all_of import InternalErrorAllOf
from cvg_sdk.model.project_health_event import ProjectHealthEvent
from cvg_sdk.model.provisioning_timed_out import ProvisioningTimedOut
from cvg_sdk.model.synthesis import Synthesis
from cvg_sdk.model.synthesis_failed import SynthesisFailed
from cvg_sdk.model.transcription import Transcription
from cvg_sdk.model.transcription_failed import TranscriptionFailed
globals()['AudioPlaybackFailed'] = AudioPlaybackFailed
globals()['EndpointCall'] = EndpointCall
globals()['EndpointCallFailed'] = EndpointCallFailed
globals()['HealthEventStatus'] = HealthEventStatus
globals()['InternalError'] = InternalError
globals()['InternalErrorAllOf'] = InternalErrorAllOf
globals()['ProjectHealthEvent'] = ProjectHealthEvent
globals()['ProvisioningTimedOut'] = ProvisioningTimedOut
globals()['Synthesis'] = Synthesis
globals()['SynthesisFailed'] = SynthesisFailed
globals()['Transcription'] = Transcription
globals()['TranscriptionFailed'] = TranscriptionFailed
from cvg_sdk.model.internal_error import InternalError


class TestInternalError(unittest.TestCase):
    """InternalError unit test stubs"""

    def setUp(self):
        pass

    def tearDown(self):
        pass

    def testInternalError(self):
        """Test InternalError"""
        # FIXME: construct object with mandatory attributes with example values
        # model = InternalError()  # noqa: E501
        pass


if __name__ == '__main__':
    unittest.main()
